requirements.txt

```diff
@@ -5,3 +5,5 @@ dask
 click
 boto3
 locket
+six
+tblib
```

Comment:
cc @koverholt I'm not sure if this is worth it, but I'm considering adding two lightweight Pure-Python dependencies to improve traceback handling.